docs/specs/clients/notify/authentication.md

```diff
@@ -121,9 +121,245 @@ A non-ideal way to avoid the race condition is for the sender to set the message
+
+### Request
+
+- act - `notify_get_notification`
+- iss - did:key of client identity key
+- ksu - key server for identity key verification
+- aud - did:key of dapp authentication key
+- app - did:web of app domain that this request is associated with 
+  - Example: `did:web:app.example.com`
+- id - the notification ID to retrieve
+
+```typescript
+{
+  auth: string,
+}
+```
+
+| IRN     |              |
+| ------- | ------------ |
+| TTL     | 300          |
+| Tag     | 4016         |
+| Topic   | notify topic |
+
+### Response
+
+- act - `notify_get_notification_response`
+- iss - did:key of client identity key
+- aud - did:key of dapp authentication key
+- nfn - [Notify Notification](./data-structures.md#notify-notification)
+
+```typescript
+{
+  auth: string,
+}
+```
+
+| IRN     |              |
+| ------- | ------------ |
+| TTL     | 300          |
+| Tag     | 4017         |
+| Topic   | notify topic |
+
+## `wc_notifyNotificationChanged`
+
+Emitted by the Notify Server when a notification changed state. For example if its read status changed.
+
+### Request
+
+- act - `notify_notification_changed`
+- iss - did:key of dapp authentication key
+- aud - did:key of client identity key
+- nfn - array of [Notify Notification](./data-structures.md#notify-notification)
+
+```typescript
+{
+  auth: string,
+}
+```
+
+| IRN     |              |
+| ------- | ------------ |
+| TTL     | 300          |
+| Tag     | 4018         |
+| Topic   | notify topic |
+
+### Response
+
+- act - `notify_notification_changed_response`
+- iss - did:key of client identity key
+- ksu - key server for identity key verification
+- aud - did:key of dapp authentication key
+
+```typescript
+{
+  auth: string,
+}
+```
+
+| IRN     |              |
+| ------- | ------------ |
+| TTL     | 300          |
+| Tag     | 4019         |
+| Topic   | notify topic |
+
+## `wc_notifyReadNotification`
+
+Marks a notification as read.
+
+### Request
+
+- act - `notify_read_notification`
+- iss - did:key of client identity key
+- ksu - key server for identity key verification
+- aud - did:key of dapp authentication key
+- app - did:web of app domain that this request is associated with 
+  - Example: `did:web:app.example.com`
+- ids - array of notification IDs to mark as read, max 1000 items
+
+```typescript
+{
+  auth: string,
+}
+```
+
+| IRN     |              |
+| ------- | ------------ |
+| TTL     | 300          |
+| Tag     | 4020         |
+| Topic   | notify topic |
+
+### Response
+
+- act - `notify_read_notification_response`
+- iss - did:key of dapp authentication key
+- aud - did:key of client identity key
+
+```typescript
+{
+  auth: string,
+}
+```
+
+| IRN     |              |
+| ------- | ------------ |
+| TTL     | 300          |
+| Tag     | 4021         |
+| Topic   | notify topic |
+
+## `wc_notifyGetUnreadNotificationsCount`
```

Comment:
Suggest removal in-favor of adding `last_sent_at` and `unread_count` to `NotifyServerSubscription`

Reply:
Migrated to new PR: https://github.com/WalletConnect/walletconnect-specs/pull/191/files#r1448068737